docs/HowToFAQ/FAQIterator.md

```diff
@@ -0,0 +1,43 @@
+---
+title: How do I create and use an iterator?
+---
+
+## Question
+
+How do I create and use an iterator?
+
+## Answer
+
+Here is an example of an iterator:
+```dafny
+iterator Gen(start: int) yields (x: int)
+  yield ensures |xs| <= 10 && x == start + |xs| - 1
```

Comment:
Does the `iterator` construct automatically create a variable `xs` to represent all of the returned values `x`? It might be good to explicitly say that if it's true.

Reply:
Yes - text added.